Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+import groovy.json.JsonOutput
+
+pipeline {
+    agent any
+    stages {
+        stage("Create Databricks cluster") {
+            steps {
+                script {
+                    def uuid = UUID.randomUUID().toString()
+                    def clusterParams = [
+                        cluster_name: "jenkins-${uuid}",
+                        spark_version: "6.2.x-scala2.11",
```

Comment:
Can we parameterize this?

Reply:
We could, but it's probably not worth doing it at this point. The jenkins machine installs databricks-connect==6.2.0, which is only compatible with DBR 6.2, so even if we wanted to test on different runtimes, it would be a big effort, eg. to create conda environments for each DBR version.